code/game/gamemodes/miniantags/revenant/revenant.dm

```diff
@@ -88,7 +88,8 @@
 			objective2.owner = src.mind
 			src.mind.objectives += objective2
 			src << "<b>Objective #2</b>: [objective2.explanation_text]"
-			ticker.mode.traitors |= src.mind //Necessary for announcing
+			//ticker.mode.traitors |= src.mind //Necessary for announcing
```

Comment:
Apparently every fucking sideantag in the game is a "traitor", in order for them to turn up on the show objectives panel I guess